app/src/main/kotlin/com/simplemobiletools/camera/activities/MainActivity.kt

```diff
@@ -404,7 +415,7 @@ class MainActivity : SimpleActivity(), PhotoProcessor.MediaSavedListener, Camera
             }
         })
 
-        camera_mode_tab.setOnTouchListener { _, event ->
+        bottom_overlay.setOnTouchListener { _, event ->
```

Comment:
Added because I was getting a crash sometimes. 
<img width="1307" alt="Screenshot 2022-11-11 at 00 12 23" src="https://user-images.githubusercontent.com/25648077/201236216-73173543-2e63-48bc-a3b8-91aa836e8bc9.png">

So swiping anywhere on the overlay should switch the mode now

Reply:
there are already some new null exceptions caught onFling() if you check master, doesnt it solve it? I couldnt reproduce it on my devices

Reply:
I saw the checks but it seems the generated code does not respect them. 
See the decompiled code

<img width="1084" alt="Screenshot 2022-11-14 at 19 34 59" src="https://user-images.githubusercontent.com/25648077/201749802-9b08f38e-44e9-4b33-8692-8d1a08988f29.png">

Reply:
Im not sure what you mean here

Reply:
The generated code removes those checks you added. So the app still crashes when any of the `MotionEvent` is null due to the `Instrinsics.checkNotNullParameter` checks